### PR TITLE
changes to num_train_tasks, data loading, and score function settings

### PR DESCRIPTION
**Pull request by tomsilver**

### CODE REVIEW

File: analysis/run_supercloud_experiments.sh [hunk 1]
```diff
@@ -5,7 +5,7 @@ NUM_SEEDS=10
 FILE="analysis/submit.py"
 
 for SEED in $(seq $START_SEED $((NUM_SEEDS+START_SEED-1))); do
-    COMMON_ARGS="--remake_data --seed $SEED"
+    COMMON_ARGS="--seed $SEED"
```
Comment (ronuchit) [inline analysis/run_supercloud_experiments.sh:8]: nice

File: src/settings.py [hunk 2]
```diff
@@ -14,6 +14,7 @@
 class GlobalSettings:
     """Unchanging settings."""
     # parameters for all envs
+    num_train_tasks = 50
```
Comment (ronuchit) [inline src/settings.py:17]: do we need to make an announcement about this? it affects amber/ashay

**Review by ronuchit — APPROVED** [review]
thank you 🙏

File: src/settings.py (hunk 2 above)
Comment (tomsilver) [reply src/settings.py:17]: yeah

**Pull request merged**